src/backend/tests/unit/test_app.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path, PosixPath
+
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from s3gw_ui_backend import NoCacheStaticFiles
+
+
+def test_static_files_1(tmp_path: PosixPath) -> None:
+    Path(tmp_path, "index.html").touch()  # Fake the file
```

Comment:
is this really faking the files? I suspect this is creating the files in the file system, and not cleaning them up.

pytest has a plugin for actual filesystem mocks, might consider using that instead.

Reply:
Or, at least, clean up?

Reply:
Or, if `tmp_path: PosixPath` is an existing fixture, maybe document that?